lizmap/lizmap-modules/localiz/install/install.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+* @package   lizmap
+* @subpackage localiz
+* @author    aeag
+* @copyright 2011 3liz
+* @link      http://3liz.com
+* @license    All rights reserved
+*/
+
+
+class localizModuleInstaller extends jInstallerModule {
+
+    function install() {
+        // installer le CSS
+        $cssFile = jApp::wwwPath('css/localiz.css');
+        if (! file_exists($cssFile)) {
+            $this->copyFile('css/localiz.css', $cssFile);
+        }
+        
+        // installer le JS
+        $jsFile = jApp::wwwPath('js/localiz.js');
+        if (! file_exists($jsFile)) {
+            $this->copyFile('js/localiz.js', $jsFile);
+        }
+
+        // conf DB
+        $localConfFile = jApp::configPath('localconfig.ini.php');
+        if (file_exists($localConfFile)) {
+            $ini = new jIniFileModifier($localConfFile);
+            $ini->setValue('db_driver', 'pgsql', 'localiz_plugin');
+            $ini->setValue('db_host', 'pgpool', 'localiz_plugin');
+            $ini->setValue('db_port', '5432', 'localiz_plugin');
+            $ini->setValue('db_database', 'refgeo2', 'localiz_plugin');
+            $ini->setValue('db_user', 'visu', 'localiz_plugin');
+            $ini->setValue('db_password', 'visu', 'localiz_plugin');
+            $ini->setValue('db_query', "select oid, geometrytype, value, label, longlabel, xmin, ymin, xmax, ymax from MYTABLE where label like '%s%%'", 'localiz_plugin');
```

Comment:
It's ok for this parameter to put it into the localconfig.ini.php

`$this->entrypoint->localConfigIni->setValue(...);`